sass/components/_categorical-display.scss

```diff
@@ -156,8 +156,10 @@
             border: 1px solid themed('CatDisplayCatContentBorder');
         }
 
-        > div:nth-child(2) {
-            border-radius: 4px 4px 0 0;
+        &.has-subcategory {
+            > div:nth-child(2) {
+                border-radius: 4px 4px 0 0;
+            }
```

Comment:
On Categorical Display, we have two kinds of sections. one of them has subcategory(Forex, ...) and another one has no sub-category(Favorite, active). For border-radius, first and the last item of each content should have half-radius and the rest should have full-radius(each box corner). I handle it by adding `has-subcategory`